taprpc/mintrpc/mint.proto

```diff
@@ -16,6 +16,19 @@ service Mint {
     */
     rpc MintAsset (MintAssetRequest) returns (MintAssetResponse);
 
+    /* tapcli `assets mint fund`
+    FundBatch will attempt to fund the current pending batch with a genesis
+    input, or create a new funded batch if no batch exists yet.
```

Comment:
Feels like we should add a bit more information about when or if this needs to be called? I'd suggest something like this: "This RPC method only needs to be called if the funding transaction (e.g. fee rate) should be customized. Otherwise SealBatch or FinalizeBatch can be called directly."

Reply:
Updated, it's narrower than that since you can provide a tapscript sibling and feerate with both `Fund` and `Finalize`. So it's really only if you plan to use `Seal` later.